lib/manageiq/api/common/paginated_response.rb

```diff
@@ -31,7 +31,7 @@ def links_hash
             "last"  => link_to_last,
             "prev"  => link_to_prev,
             "next"  => link_to_next,
-          }
+          }.compact
```

Comment:
whoa...I didn't know that would remove nil values... we should deprecate `delete_nils` from more_core_extensions.